### DISCUSSION TITLE
Welcome to lasotuvi Discussions!

### DISCUSSION THREAD

**Discussion by doanguyen** [doanguyen/lasotuvi#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dranixxxx** [comment]
chào bác, liệu bác có thể viết phần chú thích cho các sao để đưa ra lời bình giải không? cảm ơn bác
